hedera-node/hedera-admin-service-impl/src/main/java/module-info.java

```diff
@@ -18,4 +21,5 @@
     exports com.hedera.node.app.service.admin.impl.handlers;
     exports com.hedera.node.app.service.admin.impl.components;
     exports com.hedera.node.app.service.admin.impl.codec;
+    exports com.hedera.node.app.service.admin.impl.config;
```

Comment:
Why do we need to export this package? Is it used by the `hedera-app`

Reply:
Yes. TransactionDispatcher uses it in order to pass a global property (location of the zip upgrade artifact).